webapp/src/js/bootstrapper/index.js

```diff
@@ -224,53 +224,70 @@
       .catch(() => true);
 
     let isInitialReplicationNeeded;
-    Promise.all([swRegistration, testReplicationNeeded(), setReplicationId(POUCHDB_OPTIONS, localDb)])
-      .then(function(resolved) {
+    Promise
+      .all([swRegistration, testReplicationNeeded(), setReplicationId(POUCHDB_OPTIONS, localDb)])
+      .then(resolved => {
         purger.setOptions(POUCHDB_OPTIONS);
         isInitialReplicationNeeded = !!resolved[1];
 
         if (isInitialReplicationNeeded) {
+          window.startupTimes.replicationStarted = performance.now();
           return docCountPoll(localDb)
             .then(() => initialReplication(localDb, remoteDb))
             .then(testReplicationNeeded)
             .then(isReplicationStillNeeded => {
               if (isReplicationStillNeeded) {
                 throw new Error('Initial replication failed');
               }
-            });
+            })
+            .then(() => window.startupTimes.replicationEnded = performance.now());
         }
       })
       .then(() => {
         return purger
           .shouldPurge(localDb, userCtx)
           .then(shouldPurge => {
+            window.startupTimes.purgedDB = shouldPurge;
```

Comment:
`purgedDB` sounds like it stores an actual db object. I think we should change it to something more suggestive, like: `purgingRunning` or similar.